csfieldguide/static/interactives/bin-packing/README.md

```diff
@@ -0,0 +1,12 @@
+# Bin Packing Interactive
+
+**Author:** Unknown
```

Comment:
Potentially Isabelle Taylor? You could probably try checking by looking through the master branch on github